docs/source/bettertransformer/quickstart.mdx

```diff
@@ -0,0 +1,52 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
Could we move this in overview.mdx, maybe?

Reply:
Thanks ! Fixed in 9d5f90a